src/checkers/lines.rs

```diff
@@ -2,7 +2,7 @@
 
 use crate::registry::{Diagnostic, Rule};
 use crate::rules::pycodestyle::rules::{
-    doc_line_too_long, line_too_long, no_newline_at_end_of_file,
+    doc_line_too_long, line_too_long, no_newline_at_end_of_file, mixed_spaces_and_tabs
```

Comment:
I think you need to run `cargo +nightly fmt`:

```diff
❯ git diff HEAD
diff --git a/src/checkers/lines.rs b/src/checkers/lines.rs
index fd9b9aba..987e504b 100644
--- a/src/checkers/lines.rs
+++ b/src/checkers/lines.rs
@@ -2,7 +2,7 @@

 use crate::registry::{Diagnostic, Rule};
 use crate::rules::pycodestyle::rules::{
-    doc_line_too_long, line_too_long, no_newline_at_end_of_file, mixed_spaces_and_tabs
+    doc_line_too_long, line_too_long, mixed_spaces_and_tabs, no_newline_at_end_of_file,
 };
 use crate::rules::pygrep_hooks::rules::{blanket_noqa, blanket_type_ignore};
 use crate::rules::pyupgrade::rules::unnecessary_coding_comment;
```

Reply:
Done!